tutorials/colors/colors.py

```diff
@@ -5,59 +5,105 @@
-  better visibility of colored lines against typical backgrounds.
-* a X11/CSS4 color name (case-insensitive);
-* a name from the `xkcd color survey`_, prefixed with ``'xkcd:'`` (e.g.,
-  ``'xkcd:sky blue'``; case insensitive);
-* one of the Tableau Colors from the 'T10' categorical palette (the default
-  color cycle): ``{'tab:blue', 'tab:orange', 'tab:green', 'tab:red',
-  'tab:purple', 'tab:brown', 'tab:pink', 'tab:gray', 'tab:olive', 'tab:cyan'}``
-  (case-insensitive);
-* a "CN" color spec, i.e. ``'C'`` followed by a number, which is an index into
-  the default property cycle (:rc:`axes.prop_cycle`); the indexing is intended
-  to occur at rendering time, and defaults to black if the cycle does not
-  include color.
++--------------------------------------+--------------------------------------+
+| Format                               | Example                              |
++======================================+======================================+
+| RGB or RGBA (red, green, blue, alpha)| - ``(0.1, 0.2, 0.5)``                |
+| tuple of float values in a closed    | - ``(0.1, 0.2, 0.5, 0.3)``           |
+| interval [0, 1].                     |                                      |
++--------------------------------------+--------------------------------------+
+| Case-insensitive hex RGB or RGBA     | - ``'#0f0f0f'``                      |
+| string.                              | - ``'#0f0f0f80'``                    |
++--------------------------------------+--------------------------------------+
+| Case-insensitive shorthand equivalent| - ``'#abc'`` as ``'#aabbcc'``        |
+| string of RGB or RGBA from duplicated| - ``'#abcd'`` as ``'#aabbccdd'``     |
+| characters.                          |                                      |
++--------------------------------------+--------------------------------------+
+| String representation of float value | - ``'0.8'`` as light gray            |
+| in closed interval ``[0, 1]`` for    | - ``'0'`` as black                   |
+| black and white, respectively.       | - ``'1'`` as white                   |
++--------------------------------------+--------------------------------------+
+| Single character shorthand notation  | - ``'b'`` as blue                    |
+| for shades of colors.                | - ``'g'`` as green                   |
+|                                      | - ``'r'`` as red                     |
+| .. note:: The colors green, cyan,    | - ``'c'`` as cyan                    |
+|           magenta, and yellow do not | - ``'m'`` as magenta                 |
+|           coincide with X11/CSS4     | - ``'y'`` as yellow                  |
+|           colors.                    | - ``'k'`` as black                   |
+|                                      | - ``'w'`` as white                   |
++--------------------------------------+--------------------------------------+
+| Case-insensitive color name from     | - ``'xkcd:sky blue'``                |
```

Comment:
X11/CSS4 colors are missing.

and link to https://matplotlib.org/devdocs/gallery/color/named_colors.html